lib/ansible/modules/cloud/azure/azure_rm_sqldatabase.py

```diff
@@ -432,10 +428,10 @@ def create_update_sqldatabase(self):
         self.log("Creating / Updating the SQL Database instance {0}".format(self.name))
 
         try:
-            response = self.mgmt_client.databases.create_or_update(resource_group_name=self.resource_group,
-                                                                   server_name=self.server_name,
-                                                                   database_name=self.name,
-                                                                   parameters=self.parameters)
+            response = self.sql_client.databases.create_or_update(resource_group_name=self.resource_group,
```

Comment:
just wondering how this get merged  at first place..

Reply:
why?